enterprise_access/apps/api/tasks.py

```diff
@@ -94,20 +65,20 @@ def decline_enterprise_subsidy_requests_task(subsidy_request_uuids, subsidy_type
 
 
 @shared_task(base=LoggedTaskWithRetry)
-def send_notification_emails_for_requests(
-        subsidy_request_uuids,
+def send_notification_email_for_request(
```

Comment:
I like this single use version of the function much more than the multi use version as it puts the onus on the caller to properly handle any exceptions that may occur during execution. My theory being that the caller knows better how to handle a single failure within the workflow than this generic function does. 👍